packages/tlschema/api-report.api.md

```diff
@@ -1449,6 +1449,13 @@ export type TLShapePartial<T extends TLShape = TLShape> = T extends T ? {
     type: T['type'];
 } & Partial<Omit<T, 'id' | 'meta' | 'props' | 'type'>> : never;
 
+// @public (undocumented)
+export type TLShapeUpdatePartial<T extends TLShape = TLShape> = T extends T ? {
```

Comment:
For bindings, we use `TLBindingCreate` and `TLBindingUpdate` as the types for the create/update partials. Could this either match those, or could we update the binding ones to reflect this naming scheme?

Reply:
I've stumbled over those before! No preference on which we go with but I'd like us to have the same solution for both.